### ISSUE TITLE
Feedback from Morgan and Diane

### ISSUE THREAD

**Issue by msschwartz21** [dlmbl/instance_segmentation#4]
- [x] Change the "exercise" labels in the notebook to "tasks". Otherwise it's a bit confusing since the entire notebook is considered an exercise
- [ ] For the sequence of tasks that update the `TissueNetDataset` class, we think it would be easier to just give students the lines of code that they wrote in the previous task rather than asking them to copy paste. 
- [x] Move instructions and explanations for a task into markdown blocks instead of in code comments to make it easier to read
- [ ] Exercise 1.5: add a hint as to where to change the downsampling factors
- [ ] Exercise 1.6-1.7: add a link to pytorch documentations for things like nn.Sequential and casting to device, Dataloaders
- [ ] Exercise 2.3: use the actual function names in `monospace` e.g. `nn.CrossEntropy` rather than cross entropy or some people will type exactly that
- [ ] Exercise 3.4: should already have everything already done in the dataset, and just tell them where to change things
- [ ] Don’t put “checkpoint” after Bonus exercises, it makes it seem like they’re not optional ;)

---
Universal tasks
- [ ] Update environment setup to use `mamba` (See https://github.com/dlmbl/DL-MBL-2023/wiki/Exercise-Guidelines#environment)
- [ ] Add a red callout block to the beginning of each notebook that specifies which kernel to use
```
<div class="alert alert-danger">
Please use kernel <kernel name here>
</div>
```
- [ ] Check for any references to slack. We are going to use Element this year.

**Comment by pattonw** [comment]
Adapted the exercise from dl4mia (https://github.com/dl4mia/01_segmentation/blob/main/3-instance_solution.py) for this year. I think all feedback has been addressed or made irrelevant (i.e. by mamba, content etc.)
